receiver/sqlserverreceiver/README.md

```diff
@@ -14,29 +14,45 @@
 [sumo]: https://github.com/SumoLogic/sumologic-otel-collector
 <!-- end autogenerated section -->
 
-The `sqlserver` receiver grabs metrics about a Microsoft SQL Server instance using the Windows Performance Counters.
-Because of this, it is a Windows only receiver.
+The `sqlserver` receiver grabs metrics about a Microsoft SQL Server instance. The receiver works by either using the
+Windows Performance Counters, or by directly connecting to the instance and querying it. Windows Performance Counters
+are only available when running on Windows.
 
 Make sure to run the collector as administrator in order to collect all performance counters for metrics. 
 
 ## Configuration
 
 The following settings are optional:
 - `collection_interval` (default = `10s`): The internal at which metrics should be emitted by this receiver.
+- `instance_name` (optional): The instance name identifies the specific SQL Server instance being monitored.
```

Comment:
I chose to allow query-filtering based on instance name and not computer name as I believe instance name makes more sense for this. If anyone wants both, or only computer name I'm happy to discuss. It would be a relatively simple feature/enhancement.